apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -68,7 +68,10 @@ public SpannerCommand()
         {
             SpannerConnection = connection;
             _transaction = transaction;
-            Parameters = parameters;
+            if (parameters != null)
+            {
+                Parameters = parameters;
+            }
```

Comment:
how is that a compile error?  It will be just null in that case.  seems this way may be safer in that it will always be set when a new ctor is made.